controller/client/controller_client.go

```diff
@@ -473,3 +473,84 @@ func (c *ControllerClient) Check() error {
 
 	return nil
 }
+
+func (c *ControllerClient) StoreBackupReplicaMapping(backupID string, replicaAddress string) error {
```

Comment:
This is totally a random name. We cannot know which RPC function this one going to call by looking at the name. Please keep the name consistent. Same below. Use the same name as the server defined in the client.